### ISSUE TITLE
Mackup ignoring settings in .mackup.cfg

### ISSUE THREAD

**Issue by shivams** [lra/mackup#1015]
This is really surprising. `mackup` used to work just fine in my Mac. On my Linux Mint 17.1, I have created a `.mackup.cfg` file in my home folder with settings for backing up things in file_system. But somehow, `mackup` is backing up in `Dropbox`. 

I've been banging my head. The same settings worked on my Mac, but they aren't working here. It seems like it isn't reading `.mackup.cfg`.

**Comment by samuelramox** [comment]
On my Mac, mackup.cfg is not syncing the following files:

[configuration_files]
.gitignore_global
.functions
.git-completion

**Comment by lachlanhunt** [comment]
I had this problem too. As a workaround, I moved `~/.mackup.cfg` to `~/.mackup/custom.cfg`, and added an application name.

    [application]
    name = custom

    [configuration_files]
    .gitignore_global
    .git-prompt.sh
    .git-completion.bash

Then I ran `mackup backup -f` again and it backed it all up.  It does seem to be ignoring the .mackup.cfg file entirely.

**Comment by chrishiste** [comment]
@lachlanhunt Smart !! This fixed the problem for me as well 👍

**Comment by kdschaffe** [comment]
Same issue here. I tried adding 

```bash
[configuration_files]
.warprc
```
to my `.mackup.cfg` but nothing seemed to happend. @lachlanhunt solution works well enough though

**Comment by lra** [comment]
You are all right, support for this have been dropped by mistake and should be brought back again, because it's useful. See https://github.com/lra/mackup/issues/1287
